proxy/main.py

```diff
@@ -619,3 +625,20 @@ def post_deployment_set_schedule(request: Request, deployment_id, status):
         raise HTTPException(status_code=400, detail="failed to set schedule") from error
 
     return {"success": 1}
+
+
+@app.post("/proxy/flows/airbyte/reset/")
+async def reset_airbyte_conn_flow(request: Request, payload: RunAirbyteResetConnection):
+    """Prefect flow to run airbyte reset connection job"""
+    logger.info(payload)
+    if not isinstance(payload, RunAirbyteResetConnection):
+        raise TypeError("payload is invalid")
+
+    logger.info("running dbtcore-run for dbt-core-op %s", payload.slug)
+    try:
+        result = run_airbyte_conn_reset(payload)
```

Comment:
i think you want to `await` here... the reset starts fine but my http request blocked and eventually timed out

Reply:
done